slave.mk

```diff
@@ -1192,6 +1194,68 @@ $(DOCKER_LOAD_TARGETS) : $(TARGET_PATH)/%.gz-load : .platform docker-start $$(TA
 ## Installers
 ###############################################################################
 
+ifeq ($(ENABLE_RFS_SPLIT_BUILD),y)
```

Comment:
Can caching support be enabled for this artifact? IMO i think its hard as it might involve checking the versions of various packages, env variables etc. But checking with you if it is doable?

Reply:
Doable, but not trivial and out of scope